packages/commonwealth/server/routes/status.ts

```diff
@@ -69,7 +93,7 @@ const status = async (
         );
 
       return res.json({
-        chains,
+        chainsWithSnapshots,
```

Comment:
No they are still there, since what its now returning is an object that looks like `{chains: ChainInfo, snapshot: Array<string> | null}`. I've adjusted the places where we hit /status to extract the chains where needed.

Reply:
Can we make this clear by adding types to the status route?

Reply:
Type the whole route or make chainsWithSnapshots have a type?

Reply:
Type the whole route => use the TypedResponse + TypedQueryRequest stuff.